Templates/GitHubActionsPipeline/.github/workflows/Release.yml

```diff
@@ -0,0 +1,142 @@
+                required: false
+
+env:
+    uniqueWorkspaceId: /u/github/workspace/${{ github.repository }}/${{ github.ref_name }}/build_r${{github.run_number }} # adding an r to before the number,
+    # to avoid an error caused by repeated build numbers 
+    branch : ${{ github.ref_name }} 
+    application: ${{  github.event.repository.name  }}
+    pipelineType: ${{ inputs.pipelineType }}
+    packageName: ${{github.run_id}}
+    zosHostname: lpar.dev.ibm.com
+    zosSFTPUser: GITHUB@lpar.dev.ibm.com
+
+jobs:
+    Setup:
+        runs-on: self-hosted
+        steps:
+          - name: CI-CD pipeline parameter printout
+            run: |
+                echo " "
+                echo "Following are the CI-CD pipeline parameters"
+                echo "*******************************************"
+                echo "uniqueWorkspaceId                  = ${{ env.uniqueWorkspaceId}}"
+                echo "branch                             = ${{ env.branch}}"
+                echo "application                        = ${{ env.application}}"
+                echo "pipelineType                       = ${{ env.pipelineType}}"
+                echo "packageName                        = ${{ env.packageName}}"
+                echo "zosHostname                        = ${{ env.zosHostname}}"
+                echo "zosSFTPUser                        = ${{ env.zosSFTPUser}}"
+          - name: Clone Repo to z/OS Unix System Services
+            run: ssh ${{ env.zosSFTPUser}} ". ./.profile && gitClone.sh -w ${{ env.uniqueWorkspaceId }} -r https://${{ secrets.githubAccessToken }}@github.com/${{ github.repository }}.git -b ${{ env.branch }}" 
+
+    Build:
+        needs:
+            - Setup
+        runs-on: self-hosted
+        steps:
+          - name: Build git repo
+            run: ssh ${{ env.zosSFTPUser}} ". ./.profile && dbbBuild.sh -w ${{ env.uniqueWorkspaceId }} -a ${{env.application}} -b ${{ env.branch }} -p ${{env.pipelineType}}"
+
+    PrepareBuildLogs:
+        needs: 
+            - Build
+        runs-on: self-hosted
+        steps:
+          - name: Prepare Logs
+            run: |
+                ssh ${{ env.zosSFTPUser}} ". ./.profile && prepareLogs.sh -w ${{ env.uniqueWorkspaceId }}"  
+                rm -rf "${{ runner.temp }}/build-logs" 
+                mkdir "${{ runner.temp }}/build-logs" 
+                sftp ${{ env.zosSFTPUser}}:${{ env.uniqueWorkspaceId }}/*.tar "\"${{ runner.temp }}/build-logs\""
+                tar -xvf "${{ runner.temp }}/build-logs/logs.tar" -C "${{ runner.temp }}/build-logs"
+
+          - name: UploadBuildLogs
+            uses: actions/upload-artifact@v4
+            with:
+              name: build_${{github.run_number }}                        # Name used for artifact when it is uploaded
+              path: ${{ runner.temp }}/build-logs                        # Path to build logs artifacts
+              retention-days: 1
+          
+          - name: checkout
```

Comment:
@johnmnemec The checkout and the computation of the release candidate tag fits better into the package stage as a step after creating the package.

![image](https://github.com/IBM/dbb/assets/28918869/0e7d979e-7619-4cd9-9c01-e3f5110dd0f4)